sionna/channel/optical/fiber.py

```diff
@@ -0,0 +1,309 @@
+#
+# SPDX-FileCopyrightText: Copyright (c) 2021-2022 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Author: Tim Alexander Uhlemann <uhlemann@ieee.org>
+
+"""
+This module defines the split-step Fourier method to approximate the solution of
+the nonlinear Schroedinger equation.
+"""
+
+import tensorflow as tf
+from tensorflow.keras.layers import Layer
+import sionna
+from sionna.channel import utils
+
+
+class SSFM(Layer):
+    # pylint: disable=line-too-long
+    r"""SSFM(alpha, beta_2, f_c, gamma, half_window_length, ell, n_ssfm, n_sp, dt, t_norm, with_amplification, with_attenuation, with_dispersion, with_nonlinearity, swap_memory=True, dtype=tf.complex64, **kwargs)
```

Comment:
Can we set defaults values for some of the parameters (e.g, dt and t_norm)? This would simplify the usage for beginners